src/db/connection.cr

```diff
@@ -31,7 +31,11 @@ module DB
     # :nodoc:
     property auto_release : Bool = true
 
+    @created_at : Time
+    getter created_at
+
     def initialize(@context : ConnectionContext)
+      @created_at = Time.utc
```

Comment:
I added this + the getter initially for some very helpful debugging information as I was trying to address system-level disconnects and figure out when they were beginning to happen; I'm not using it currently and could remove it, but it's turned out to be useful enough that I opted to leave it for review as a possible useful addition.

Reply:
👍